src/components/attendee/NavOverlay/ScheduleOverlay/ScheduleOverlay.tsx

```diff
@@ -69,116 +82,163 @@ export const ScheduleOverlay: React.FC = () => {
   const isScheduleTimeshifted = !isToday(firstDayOfSchedule);
 
   const weekdays = useMemo(() => {
-    if (dayDifference <= 0) return ALWAYS_EMPTY_ARRAY;
-
-    return range(dayDifference)
-      .map((dayIndex) => {
-        const eventDay = addDays(firstScheduleDate, dayIndex);
-
-        const daysWithEvents = liveAndFutureEvents.filter(
-          isEventWithinDateAndNotFinished(eventDay)
-        );
+    const formatDayLabel = (day: Date | number) => {
+      if (isScheduleTimeshifted) {
+        return format(day, "do");
+      } else {
+        return formatDateRelativeToNow(day, {
+          formatOtherDate: (dateOrTimestamp) => format(dateOrTimestamp, "do"),
+          formatTomorrow: (dateOrTimestamp) => format(dateOrTimestamp, "do"),
```

Comment:
There should be constants for this in `dateSettings.ts` with the proper library name as prefix, keeps it easier to track all the different formats we use.

Reply:
replaced with a const from utils [04e9125](https://github.com/sparkletown/sparkle/pull/2840/commits/04e91251fb1d5bfc42be24e1adab66087443d930)